app/i18n.js

```diff
@@ -17,9 +17,9 @@ export const appLocales = [
 addLocaleData(enLocaleData);
 
 export const formatTranslationMessages = (locale, messages) => {
-  const defaultFormattedMessages = locale !== DEFAULT_LOCALE
```

Comment:
Dont feel that strongly about it. Thought it might be more readable to not have negated conditions